web-registry/src/pages/ProjectReview/ProjectReview.util.tsx

```diff
@@ -86,5 +105,37 @@ const getCountryKey = (country: string): string => {
     );
   });
   if (foundKey) return foundKey;
+
+  // TODO: iso3166 did not pick up these US variants. May need to add more data sources.
+  const isUnitedStates = /United States|USA|U.S./.test(country);
```

Comment:
This is a band-aid. The iso3166 library has US as "United States of, the" or something ridiculous like that, so I had to handle some other possibilities. I think it might be good to bring back the countries.ts file, so we can handle edge cases manually.

Reply:
mmm, I don't know exactly what the problem was... but I just checked the `iso-3166-2` library and US only appears with the name United States

It can be easily checked here:

https://github.com/olahol/iso-3166-2.js/blob/master/data.csv

Reply:
I'll have to see if I can reproduce. Maybe there was just one case where I ran into this.